.travis.yml

```diff
@@ -2,7 +2,9 @@ language: java
 jdk:
   - oraclejdk8
 addons:
-  sauce_connect: true
+  sauce_connect:
+    username: alloyui
```

Comment:
@fernandosouza no, the tests for ios were just not running for some reason, so maybe it's because of some security dialog like this. Maybe this `no_ssl_bump_domains` fix will also work for ios, so it's worth trying it out after this I guess.